lib/pkgcloud/core/network/securityGroup.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * securityGroup.js: Base securityGroup from which all pkgcloud securityGroup inherit.
+ *
+ * (C) 2015 Rackspace
+ *      Shaunak Kashyap
+ * MIT LICENSE
+ */
+
+var util = require('util'),
+    model = require('../base/model');
```

Comment:
why not make this `Model = require('../base/model').Model` so you don't have to reference it everywhere?